include/crm/common/remote_internal.h

```diff
@@ -61,6 +61,20 @@ pcmk__is_guest_or_bundle_node(const pcmk_node_t *node)
            && (node->details->remote_rsc->container != NULL);
 }
 
+/*!
+ * \internal
+ * \brief Check whether a node is a Pacemaker Remote node of any kind
+ *
+ * \param[in] node  Node to check
+ *
+ * \return true if \p node is a remote, guest, or bundle node, otherwise false
+ */
+static inline bool
+pcmk__is_pacemaker_remote_node(const pcmk_node_t *node)
+{
+    return (node != NULL) && (node->details->type == pcmk_node_variant_remote);
```

Comment:
Could call this in `pcmk__is_{remote,guest_or_bundle}_node()` to reduce duplication, even though the code isn't much shorter that way

Reply:
That is more readable